### ISSUE TITLE
[Bug] `Date` serialized without a key in output object-string.

### ISSUE THREAD

**Issue by jakerdy** [j-brooke/FracturedJson#31]
Hi, I been playing with different serialization libs, and found little bug in FracturedJson.

When I tried to serialize object that contains `Date` value, I got object with date value enclosed in "", but key for this value didn't printed into output string...

Reproducible exmaple:

```typescript
import { Formatter } from "fracturedjsonjs";

const obj = {
    num: 123,
    str: "abc",
    date: new Date(),
};

const fjson = new Formatter();

console.log(fjson.Serialize(obj));
```

Output:
```
{"num": 123, "str": "abc", "2024-04-28T16:41:05.205Z"}
```

As you can see, there is no `"date": ` before the date.

Tried this with different settings. Formating changing, but key is still missing.

My environment:
- Node: 20.10.0
- fracturedjsonjs: 4.0.0

**Comment by j-brooke** [comment]
Thanks for reporting this.  I'll have a fix soon.

**Comment by j-brooke** [comment]
Fixed in fracturedjsonjs v4.0.1.  (This problem only applies to the JS library - not any of the others.)
